### PR TITLE
Made it easy to download and start using this project

### LINKED ISSUE #4: Make it easy to download a clean project (for starting a new project)
Use Raygun for generating new projects:
https://github.com/carbonfive/raygun-rails

In addition to changing all the names, it may also need to:
- delete example files
- modify the README.md to not be Roadrunner Rails

### PR DESCRIPTION
**Pull request by plusjeff**
Goals of this:

- Be able to download this as a ZIP file and get started
- Instructions provided on how to use this repository (in README)
- No uncommented sample code
- Clean README file for getting started

In the future, we should:

- Fill in the TODO items in the README.md
- Add in instructions on how to get started (seen at the top of the README) to the Road Runner guide
- Add a download ZIP link from the Road Runner guide

This closes #4 . It's in the spirit of Raygun, without needing to use Raygun.

### CODE REVIEW

**Comment by plusjeff** [comment]
P.S. @shanear Sorry for removing the ASCII art :crying_cat_face: ! It just, well, was in the way.

**Comment by MarinaMartin** [comment]
Why would I download this as a zip?

File: README.md [hunk 1]
```diff
@@ -1,54 +1,44 @@
...
-      J4wwLagagagWWWWWhwilld                                     ______________  | '__/ _` | | / __|
-         ?!4m#m####ZZ#Zmlllf               p           ------------------------  | | | (_| | | \__ \
-      gKX@CiillTYXmDYYTlllmp         aggQ"4XXLga               ________________  |_|  \__,_|_|_|___/
-    aGZF????????    )?4@illf   aggJQ"!'=jg#?':?"4#Lgga
-  aAq"'.                "wuRXXXm!?     ]X#Xp      J!mX#XZZXUa
-aAm?                     )!"!'.         !pXP           !XXZXXQ
-r'                                                      )4XXWW
-.
-```
-
```
Comment (shanear) [inline README.md]: Noooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooo :cry: :cry: :cry: :cry: :cry: :cry: :cry: :cry: :cry: :cry: :cry: :cry: :cry: :cry: :cry: :sob: :sob: :sob: :sob: :sob: :sob: :sob: :sob: :sob: :sob: :sob: :sob: :sob: :sob: :sob: :sob:

**Comment by shanear** [comment]
why not remove all irrelevant stuff in a startup script? This script can either be part of a `./bin/init` or a gem that downloads this file and sets everything up (like our own raygun).

What do you think about that?

**Comment by plusjeff** [comment]
@MarinaMartin : Why wouldn't you download this as a ZIP? :smile: Is there another way you would propose folks get this project, so that they can start building something?

@shanear : I'm cool with that. My thought was that it would simply be less to maintain, and examples could go into the guide. If you have strong feelings on the init script, let's go that route, since I'm ambivalent as long as folks can follow some quick instructions to get going.

**Comment by plusjeff** [comment]
@shanear and I talked offline. Let's junk this PR in favor of the init script. He had some good ideas around making the init script flexible to add things only for specific kinds of projects.

**Comment by MarinaMartin** [comment]
@plusjeff Nevermind, I guess git clone is basically that.

**Pull request closed without merging**